app/assets/javascripts/device-time.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function(){
+  function setElementTime() {
+    el.textContent = moment().format('HH:mm');
```

Comment:
there's no bound to the correct context, el may be undefined
either pass as argument or query within a func body